src/herbivoresAndCarnivores.js

```diff
@@ -1,16 +1,45 @@
 'use strict';
 
 class Animal {
-  // write your code here
-}
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+  };
+};
+
+Animal.alive = [];
 
 class Herbivore extends Animal {
-  // write your code here
-}
+  constructor(name, health, hidden) {
+    super(name);
+
+    if (hidden === undefined) {
+      this.hidden = false;
+    };
```

Comment:
all Herbivores are hidden by default

```suggestion
  constructor(name, health) {
    super(name, health);
    this.hidden = false;

```